### ISSUE TITLE
The German language file contains some phrases instead of words only

### ISSUE THREAD

**Issue by andy5995** [RazorSh4rk/random-word-api#28]
I thought about doing a PR but I think the diff might be virtually impossible to review. Is there a way to use regex to remove the phrases? I searched de.json and found 7008 spaces.

Maybe instead of editing the json, the API could filter out phrases?

Shows one use-case

https://github.com/theimpossibleastronaut/aawordsearch/pull/8#issuecomment-1330062022

Thanks for implementing additional languages, btw. :)

**Comment by RazorSh4rk** [comment]
I will look into it, could you filter spaces on the frontend until then?

**Comment by andy5995** [comment]
Yeah, I did this for now https://github.com/theimpossibleastronaut/aawordsearch/blob/6c57da7eadbeaeac3823ce15ee7e01f4830b6d3d/aawordsearch.c#L642
